src/main/java/blackjack/user/PlayerName.java

```diff
@@ -12,20 +14,34 @@ public PlayerName(final String name) {
     }
 
     private void validateNameSize(final String name) {
-        if (name.isEmpty()) {
+        if (name.isBlank()) {
             throw new IllegalArgumentException("이름은 한 글자 이상 입력해야 합니다.");
         }
     }
 
     private void validateCharacter(final String name) {
-        boolean isNotAllLetter = name.chars().anyMatch(ch -> !Character.isLetter(ch));
+        boolean isNotAllLetter = name.chars().anyMatch(c -> !Character.isLetter(c));
 
         if (isNotAllLetter) {
             throw new IllegalArgumentException("이름은 영어/한글만 입력 가능합니다.");
         }
     }
 
-    public String getName() {
+    public String getText() {
         return name;
     }
+
+    @Override
+    public boolean equals(Object object) {
+        if (object == null || getClass() != object.getClass()) {
+            return false;
+        }
+        PlayerName that = (PlayerName) object;
+        return Objects.equals(name, that.name);
+    }
```

Comment:
```suggestion
    @Override
    public boolean equals(Object o) {
        if (!(o instanceof PlayerName that)) return false;
        return Objects.equals(name, that.name);
    }
```
equals 오러라이딩을 자동 생성하면 2가지 방식 중 선택할 수 있는데요, 저는 간결함을 이유로 위의 방식을 주로 애용해요! 사나는 어떻게 생각하시는지 궁금해요~
<img width="443" alt="스크린샷 2025-03-15 12 11 00" src="https://github.com/user-attachments/assets/00f36878-b2fe-401e-a750-b0a920f30ee8" />

Reply:
훨씬 간결해지는 방법인 것 같습니다! 🙂